lib/php/lib/Transport/TTransport.php

```diff
@@ -22,6 +22,7 @@
 
 namespace Thrift\Transport;
 
+use Thrift\Exception\TTransportException;
```

Comment:
Why is this change needed?

Reply:
There is doc comment `@throws TTransportException` at Line 45.

No one (and IDE) knows which TTransportException.